.github/workflows/documentation.yml

```diff
@@ -8,7 +8,10 @@ on:
       - '*.md'
       - '.github/workflows/documentation.yml'
 
-permissions: {}
+permissions: 
+  contents: write
+  pull-requests: write
+
 
 jobs:
   docs:
```

Comment:
You can set permissions inside `docs:` like so:
```
  docs:
      permissions:
        contents: write
        pull-requests: write
```
@connormaglynn did some recent investigation where he identified that only `contents: write` should be required for this action, but we allow access to `pull-requests` in other repositories.

Reply:
created permissions under doc's and omitted pull-requests